Update monitoring-data-sources.md

nromyn opened the pull request:
Link to Activity Log Events 404'd - replaced with a suggestion for the correct link

rmca14 commented:
Date change made in another pull request, which you received and must approve. In the future, please feel free to change the date yourself and submit a pull request -- and that pull request will then merged by a pull request approver. Thanks.

nromyn committed:
Update monitoring-data-sources.md

Link to Activity Log Events 404'd - replaced with a suggestion for the correct link
